DigitalLearningSolutions.Web/Helpers/CentreCustomPromptHelper.cs

```diff
@@ -196,8 +202,29 @@ public static string GetDelegateCustomPromptAnswerName(int customPromptNumber)
                 4 => nameof(DelegateUserCard.Answer4),
                 5 => nameof(DelegateUserCard.Answer5),
                 6 => nameof(DelegateUserCard.Answer6),
-                _ => throw new ArgumentOutOfRangeException()
+                _ => throw new ArgumentOutOfRangeException(),
             };
         }
+
+        public static List<(int id, string name)> MapCentreCustomPromptsToSelectList(

```

Comment:
Technically, we are not returning something that is specifically a select list. Maybe worth renaming this to
`MapCentreCustomPromptsToDataForSelectList`?

Reply:
Good idea, changed to that name